slave/url_creator.py

```diff
@@ -60,6 +60,40 @@ def latest(self, **kwargs):
         url = re.findall("https://builds-nightly.webkit.org/files/trunk/mac/WebKit-SVN-r[0-9]*.dmg", html)
         return url
 
+
+class TaskClusterIndexHelper(object):
+    _index_url = 'https://index.taskcluster.net/v1/task/gecko.v2'
+    _artifacts = 'https://public-artifacts.taskcluster.net'
+
+    @classmethod
```

Comment:
(or maybe it won't work, because these methods refer to `_index_url` et al below?)